lib/phlex/translation.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module Phlex
+	module Translation
+		def self.included(view)
+			view.extend(ClassMethods)
+		end
+
+		module ClassMethods
+			attr_writer :translation_path
+
+			def translation_path
+				@translation_path ||= begin
+					source_location = const_source_location(name).first # Get view absolute file path
+					relative_path = source_location.split(Phlex.configuration.phlex_path).last # Get relative path to the phlex path
+					relative_path.delete_prefix("/").delete_suffix(".rb").tr("/", ".")
```

Comment:
The suggestion was to use `#[]`, but i ended up doing it another way. If you have a suggestion on how to improve this using `#[]` i will accept it.